x-pack/plugins/security_solution/cypress/e2e/dashboards/detection_response.cy.ts

```diff
@@ -109,11 +106,7 @@ describe('Detection response view', () => {
     });
   });
 
-  context('Redirection to AlertPage', { testIsolation: false }, () => {
-    afterEach(() => {
-      navigateFromHeaderTo(DETECTION_RESPONSE);
-    });
-
+  context('Redirection to AlertPage', () => {
```

Comment:
Why is test isolation removed in this suite?

Reply:
`testIsolation` is an antipattern that never should be used again 
https://docs.cypress.io/guides/core-concepts/test-isolation#What-is-Test-Isolation